handlers/amazon_s3_file_upload_from_submission_v2/handler/init.rb

```diff
@@ -23,91 +23,255 @@ def initialize(input)
     @debug_logging_enabled = ["yes","true"].include?(@info_values['enable_debug_logging'].downcase)
     puts "Parameters: #{@parameters.inspect}" if @debug_logging_enabled
 
-    @region = !@info_values["region"].to_s.empty? ? @info_values["region"] : @parameters["region"]
+    @region = !@info_values["region"].to_s.empty? ? 
+      @info_values["region"] : 
+      @parameters["region"]

```

Comment:
Same as the other handler - isn't this logic backwards?

Reply:
Can you explain?  I see If info_values: region is not empty use it or use the parameter.

Are we thinking that if the parameter is provided use it and if not fall back to the info values?

Reply:
Yes I believe the @info_values are supposed to be defaults, and the @parameters values are meant to override the defaults. So would be:

`@region = !@parameters["region"].to_s.empty? ? @parameters["region"] : @info_values["region"]`